src/main/java/org/generationcp/middleware/liquibase/UpdateLabelPrintingPresetsTask.java

```diff
@@ -0,0 +1,130 @@
+package org.generationcp.middleware.liquibase;
+
+import com.fasterxml.jackson.databind.MapperFeature;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import liquibase.database.Database;
+import liquibase.database.jvm.JdbcConnection;
+import liquibase.exception.CustomChangeException;
+import liquibase.exception.DatabaseException;
+import liquibase.exception.SetupException;
+import liquibase.exception.ValidationErrors;
+import liquibase.resource.ResourceAccessor;
+import org.generationcp.middleware.domain.labelprinting.LabelPrintingPresetDTO;
+import org.generationcp.middleware.domain.labelprinting.PresetDTO;
+
+import java.io.IOException;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class UpdateLabelPrintingPresetsTask implements liquibase.change.custom.CustomTaskChange {
+
+	private static final Integer MAX_FIELD_STATIC_ID = 63;
+	private ObjectMapper jacksonMapper;
+
+	@Override
+	public void execute(final Database database) throws CustomChangeException {
+
+		final List<String> tool_sections = Arrays.asList(
+			"DATASET_LABEL_PRINTING_PRESET", "LOT_LABEL_PRINTING_PRESET",
+			"GERMPLASM_LABEL_PRINTING_PRESET", "GERMPLASM_LIST_LABEL_PRINTING_PRESET",
+			"STUDY_ENTRIES_LABEL_PRINTING_PRESET");
+
+		this.jacksonMapper = new ObjectMapper();
+		this.jacksonMapper.disable(MapperFeature.DEFAULT_VIEW_INCLUSION);
+		JdbcConnection dbConn = null;
+		Statement selectStatement = null;
+		Statement updateStatement = null;
+
+		try {
+			dbConn = (JdbcConnection) database.getConnection();
+			dbConn.setAutoCommit(false);
```

Comment:
@cuenyad is this needed? I just read that it was false by default

Reply:
done!